scripts/schemas.js

```diff
@@ -66,5 +66,4 @@ const main = async () => {
   }
 }
 
-// this is a main function
```

Comment:
so this isn't a main function anymore? :joy:

Reply:
Not anymore. But next time we need to force the rerun of schema generation, it will definitely be the main function again